include/opentxs/blockchain/crypto/AddressStyle.hpp

```diff
@@ -17,5 +17,6 @@ enum class AddressStyle : std::uint16_t {
     P2WPKH = 3,
     P2WSH = 4,
     P2TR = 5,
+    ChecksummedHex = 6,
```

Comment:
I have used name `ChecksummedHex` because it seems to me like a kinda proper name
![image](https://user-images.githubusercontent.com/15064484/186096686-fe194df3-2719-44fe-87a8-8befc7022504.png)
https://eips.ethereum.org/EIPS/eip-2304